pic-sure-ui/src/main/webapp/picsureui/picSure/queryBuilder.js

```diff
@@ -1,6 +1,10 @@
-define([], function(){
-	var createWhere = function(pui, logicalOperator, constrainByValue, constrainParams){
-		var whereQuery = {
+define(['underscore'], function(_){
+    // The default behavior. Since we are overriding the module as a whole, we have no choice but to duplicate the code here.
+    var createWhere = function(pui, logicalOperator){
+        if(pui.includes('GRIN_CONSENT')||pui.includes('BIOBANK_FLAG')){
```

Comment:
I don't like this. Add yourself a JIRA ticket to generalize this, or add it as an override